website/src/lib/wordpress.js

```diff
@@ -370,37 +374,49 @@ export async function getCategories() {
  * (Might merge into other queries via fragment)
  * */
 export async function getGlobalProps() {
+  let fragmentMenu = /* GraphQL */ `
+    nodes {
+      id
+      databaseId
+      label
+      parentId
+      url
+      path
+    }
+`;
+
   let query = /* GraphQL */ `
     fragment Menus on RootQuery {
-      menus {
-        nodes {
-          id
-          databaseId
-          name
-          menuItems {
-            nodes {
-              id
-              label
-              parentId
-              url
-              path
-            }
-          }
-        }
+      menuHeader: menuItems(where: { location: HEADER }, first: 100) {
+        ${fragmentMenu}
+      }
+      menuFooter: menuItems(where: { location: FOOTER }, first: 100) {
+        ${fragmentMenu}
+      }
+      menuFooterSocial: menuItems(where: { location: FOOTER_SOCIAL }, first: 100) {
+        ${fragmentMenu}
+      }
+      menuFooterSecondary: menuItems(where: { location: FOOTER_SECONDARY }, first: 100) {
+        ${fragmentMenu}
       }
     }
 
+    # fragment GlobalOptions on RootQuery {
```

Comment:
Should this remain commented out?

Reply:
Yeah, we don't actually ship with global options, so this is more sample code than something we want live. (also was commented out before)